### ISSUE TITLE
Resolver Conf Defaults to Google DNS.

### ISSUE THREAD

**Issue by Smithx10** [joyent/mi-centos-bhyve-7#2]
After deploying the latest Centos-7 Image onto Triton I noticed It defaulted to the google resolvers, 8.8.8.8, 8.8.4.4.

CentOS Image with the issue: 1f11188a-c71c-11e8-83d4-370e2a698b16

Id expect this to only have my defined network resolvers.  This also slows down provisioning time if they aren't available.

Note:  
  Ubuntu 16.04  (6b47e1d9-36b8-4b6f-8764-5ff5fe6d120b)
  Ubuntu 18.04 (ac99517a-72ac-44c0-90e6-c7ce3d944a0a)
Both Work as expected.

```
#### Create Machine
[Sun 19/01/13 21:35 EST][s002][x86_64/darwin17.3.0/17.7.0][5.4.2]
<bruce.smith@cbgw3017877:~/go/src/github.com/prometheus/prometheus/discovery>
zsh/2 10270  (git)-[triton_gz]-% itt create 1f11188a-c71c-11e8-83d4-370e2a698b16 sample-2G -N sdc_nat -N My-Fabric-Network -n test-centos
Creating instance test-centos (78eddab3-199a-49f4-ab63-e804488dccde, centos-7@20181003)

#### SSH In
[Sun 19/01/13 21:35 EST][s002][x86_64/darwin17.3.0/17.7.0][5.4.2]
<bruce.smith@cbgw3017877:~/go/src/github.com/prometheus/prometheus/discovery>
zsh/2 10271  (git)-[triton_gz]-% itt ssh
[Sun 19/01/13 21:36 EST][s002][x86_64/darwin17.3.0/17.7.0][5.4.2]
<bruce.smith@cbgw3017877:~/go/src/github.com/prometheus/prometheus/discovery>
zsh/2 10271 [130]  (git)-[triton_gz]-% itt ssh centos-test

### Cat Resolvers
[root@test-centos ~]# cat /etc/resolv.conf
; Created by cloud-init on instance boot automatically, do not edit.
;
# Generated by NetworkManager
nameserver 8.8.8.8
nameserver 8.8.4.4
nameserver myresolver1
[root@test-centos ~]# mdata-get sdc:resolvers
["myresolver1","myresolver2"]
```

**Comment by mgerdts** [comment]
Can you provide the uuid of the image that is not working as expected?

**Comment by Smithx10** [comment]
CentOS Image with the issue: 1f11188a-c71c-11e8-83d4-370e2a698b16

**Comment by dcrudgington** [comment]
I just pushed out a new image (d6e56c0e-1e51-11e9-8ff1-2fecf38a5566) which includes this fix and update to CentOS 7.6.  Thanks

**Comment by Smithx10** [comment]
Thank you!!!
